ui/app/serializers/transform.js

```diff
@@ -0,0 +1,20 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    if (payload.data.masking_character) {
+      payload.data.masking_character = String.fromCharCode(payload.data.masking_character);
+    }
+
+    if (payload.data.templates && payload.data.templates.length === 1) {
+      // add space after comma in returned array length one of a string of templates
+      payload.data.templates[0] = payload.data.templates[0].replace(/,/g, ', ');
```

Comment:
If we're adding a comma for display purposes I would expect the controller to add that in, not the serializer. I'm also a little confused why we're adding it only to the first template? Maybe an example of the before and after here would be helpful.

Reply:
Let's discuss during our sync.  This was a request by design, but becomes a problem with Roles because the data is structured differently.  And yeah, I originally thought of using a serializer because it was changing the data, but it's more of a display thing.  It should go in a controller.

Reply:
Removing for now based on discussion